build/deps/markdownlint.Dockerfile

```diff
@@ -1 +1,4 @@
 FROM ghcr.io/igorshubovych/markdownlint-cli:v0.32.1
+ENTRYPOINT [""]
+
+RUN npm install --location=global textlint textlint-rule-one-sentence-per-line
```

Comment:
Let's create a separate Dockerfile for that linter. Is there a pre-built Docker image we could use?

Reply:
Textlint doesn't have Docker image, but we can create our own one and install only rules which we need. I will do it now

Reply:
Also, https://github.com/IQTLabs/textlint-rule-one-sentence-per-line (btw, check out [this issue](https://github.com/IQTLabs/textlint-rule-one-sentence-per-line/issues/7) ;)) is archived. Is there some alternative?

Reply:
@AlekSi I've checked some alternative solutions, in my opinion [this one](https://github.com/bobheadxi/readable) could be good, but my try to make it work with docker wasn't successful.